docs/docs/quickstart.mdx

```diff
@@ -82,22 +72,31 @@ const needsLogin = (req, res, next) => {
       res.redirect('/login');
     });
 };
+
+// You can use `needsLogin` as a middleware for Express: 
+// app.get("/dashboard", needsLogin, dashboard)
```

Comment:
I think the original example is clearer ("where does `app` come from?")

Reply:
I thought about that, too, but I think the spirit of this comment is just to show "it can be used as middleware later" and not necessarily to show people how to import modules from Express. Since we're already assuming basic knowledge of JavaScript / Express, I thought it would be safe to remove boilerplate code just to highlight the API usage itself.

If you disagree, I can revert this (not a huge deal in my opinion).

Reply:
I'd prefer to have this reverted. Explicit documentation is better than implicit. No room for interpretation of things but clear, concise technical instructions. You can comment `import from express` with ("or any other web framework")